### PR TITLE
Update select component to work with :global and get freestyle to work

### PR DESCRIPTION
**Pull request by tintinthong**
This PR is intended to get everything minimally working. A lot of the things were not working as I expected. Here are a few UI changes that you'll find. Note: wormhole is buggy when using animations and I will check that out separately.

In this PR, we also use :global to solve the issues where the styles cannot be applied to dropdown element which is teleported in the wormhole

Screenshot 2024-04-22 at 17 34 47

Screenshot 2024-04-22 at 17 36 56

Screenshot 2024-04-22 at 17 50 28

Screenshot 2024-04-22 at 17 37 14

**Before**

Screenshot 2024-04-22 at 15 57 04

**After**

Screenshot 2024-04-22 at 17 44 49

### CODE REVIEW

File: packages/boxel-ui/addon/src/components/input-group/accessories/index.gts [hunk 1]
```diff
@@ -211,6 +204,14 @@ export const Select: TemplateOnlyComponent<SelectAccessorySignature> =
         border-color: var(--boxel-error-100);
       }
     </style>
+    <style>
```
Comment (backspace) [inline packages/boxel-ui/addon/src/components/input-group/accessories/index.gts]: Does this need to be in a separate `style` tag?

**Review by backspace — APPROVED** [review]
Is it a design mandate that the dropdown items should be bold now? It looks unexpected to me 🤔

**Review by jurgenwerk — APPROVED** [review]

**Review by lukemelia — CHANGES_REQUESTED** [review]
Is this what you're seeing? If so, the layout as it relates to the down arrow should be fixed before merging:

image

File: packages/boxel-ui/addon/src/components/input-group/accessories/index.gts (hunk 1 above)
Comment (tintinthong) [reply packages/boxel-ui/addon/src/components/input-group/accessories/index.gts]: No it doesn

**Comment by tintinthong** [comment]
> Is this what you're seeing? If so, the layout as it relates to the down arrow should be fixed before merging:
> 
> image

Do you mean that the arrow should be justified right on the trigger (tht shud be the default ember power select)?  

https://github.com/cardstack/boxel/pull/1187/commits/b0729df640f4fbc7c84d95341140233ef00deda3

Screenshot 2024-04-23 at 10 58 08

**Comment by tintinthong** [comment]
This is the new current layout 

Screenshot 2024-04-24 at 14 51 30

**Comment by tintinthong** [comment]
I can see a regression due of @matchTriggerWidth=true . Originally, filed here

- https://github.com/cardstack/cardstack/pull/3339
- https://github.com/cibernox/ember-power-select/issues/301
- https://github.com/cibernox/ember-power-select/issues/1035 here it expresses that a solution is very difficult

The default is actually for this to be true in ember-power-select

id rather pass the argument thru than commit to such a decision of defaulting to false. I think ppl will find the fact that the dropdown doesn't span the trigger more awkward. If they dont want their content to crop then give the card maker the option to set to false.

**Pull request merged**